airqo-platform/AirQo-frontend
# Add all available docs

#### Summary of Changes (What does this PR do?)
- Enhances API documentation
- Adds Mobile app documentation
- Adds platform documentation

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
* Check out the [pr-preview link](https://ft-otherdocs-docs-preview-w7kzhvlewq-ew.a.run.app) provided in the comment by the GitHub bot.
* But if you really want to run it locally, run `yarn run docker:build` to build the application
* Run `yarn run docker:cli` to run the application
* Checkout http://localhost:3000/. You should see all the changes mentioned in the summary above.
* After you are done, run `yarn run docker:clean` do remove the residue containers.

#### What are the relevant tickets?
- [AN-62](https://airqoteam.atlassian.net/browse/AN-62?atlOrigin=eyJpIjoiMzAwNmE1Y2I2ODg0NDdkNGJkOGM1ODE5NzAzYjIwMzQiLCJwIjoiaiJ9)

#### Screenshots (optional)
![image](https://user-images.githubusercontent.com/41787587/193867903-89e59583-7645-4e00-8e54-9752ffd4d094.png)
